database/seeders/SettingSeeder.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace Database\Seeders;
+
+use Illuminate\Database\Console\Seeds\WithoutModelEvents;
+use Illuminate\Database\Seeder;
+use App\Setting;
+
+class SettingSeeder extends Seeder
+{
+    /**
+     * Run the database seeds.
+     *
+     * @return void
+     */
+    public function run()
+    {
+        //standard settings from the production site
+        $setting = new Setting(['name' => Setting::SETTING_BLOCKED_EMAIL_DOMAINS, 'type' => Setting::TYPE_STRING, 'value' => 'student.tue.nl;student.fontys.nl;tue.nl']);
+        $setting->save();
```

Comment:
I'd suggest creating all settings references and collecting them in a set or list without saving them to the DB yet. Then constructing a query to save all at once at the end.